### PR TITLE
[6137] XB - item actions fail when performed over empty sections (not stored in model yet)

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/6137

### CODE REVIEW

File: ui/app/src/services/content.ts [hunk 1]
```diff
@@ -626,7 +626,15 @@ export function moveItem(
           targetParentPath,
           (element) => {
             const item: Element = extractNode(element, targetFieldId, targetIndex);
-            const field: Element = extractNode(element, targetFieldId, removeLastPiece(`${targetIndex}`));
+            let field: Element = extractNode(element, targetFieldId, removeLastPiece(`${targetIndex}`));
+
+            // If field doesn't exist yet in the document, create it
+            if (!field) {
```
Comment (jvega190) [inline ui/app/src/services/content.ts:637]: @rart need to discuss this

File: ui/app/src/services/content.ts [hunk 2]
```diff
@@ -626,7 +632,14 @@ export function moveItem(
           targetParentPath,
           (element) => {
             const item: Element = extractNode(element, targetFieldId, targetIndex);
-            const field: Element = extractNode(element, targetFieldId, removeLastPiece(`${targetIndex}`));
+            let field: Element = extractNode(element, targetFieldId, removeLastPiece(`${targetIndex}`));
+            // If field doesn't exist yet in the document, create it
+            if (!field) {
+              const newField = createElement(originalFieldId);
+              newField.setAttribute('inline', 'true');
```
Comment (rart) [inline ui/app/src/services/content.ts]: The `inline="true"` attribute is only for the `item` element of **embedded components** of a node selector type **field**. Did you mean `item-list="true"`? Are you creating a field or an item?

File: ui/app/src/services/content.ts [hunk 3]
```diff
@@ -597,7 +597,13 @@ export function moveItem(
       const targetModelElement =
         parentDocumentModelId === targetModelId ? element : element.querySelector(`[id="${targetModelId}"]`);
       const item = extractNode(sourceModelElement, originalFieldId, originalIndex);
-      const targetField = extractNode(targetModelElement, targetFieldId, removeLastPiece(`${targetIndex}`));
+      let targetField = extractNode(targetModelElement, targetFieldId, removeLastPiece(`${targetIndex}`));
+      if (!targetField) {
+        const newField = createElement(originalFieldId);
+        newField.setAttribute('inline', 'true');
```
Comment (rart) [inline ui/app/src/services/content.ts]: As above.

**Review by rart — CHANGES_REQUESTED** [review]

File: ui/app/src/services/content.ts (hunk 2 above)
Comment (jvega190) [reply ui/app/src/services/content.ts]: Yeah I'm creating a field, I got confused about the attribute name. Fixing

**Review by rart — APPROVED** [review]

**Pull request merged**